lib/diagnostic.rb

```diff
@@ -67,8 +91,9 @@ def initialize
 
 ##
 # your answers here
-Answer.housecat_noise = nil
-#
+Answer.housecat_noise = 'I am a HouseCat, and I go 'meow''
+self refers to the housecat class, and sound is what it is initialized with. Also I ran it in pry and that is what it came out with
+
```

Comment:
You need to use double quotes when nesting single quotes, otherwise it will throw any error. And you forgot to comment your description